Corona-Warn-App/src/test/java/de/rki/coronawarnapp/risk/RiskLevelTaskTest.kt

```diff
@@ -86,4 +90,17 @@ class RiskLevelTaskTest : BaseTest() {
 //
 //        verify { riskLevelSettings.lastUsedConfigIdentifier = "config-identifier" }
```

Comment:
on purpose? Test is now empty.

Reply:
Basically an unrelated TODO from the RiskLevelTask refactoring. We should increase test coverage for this task in another PR.